src/System.Drawing.Primitives/src/System/Drawing/Size.cs

```diff
@@ -190,5 +240,25 @@ public int Height
         ///    </para>
         /// </summary>
         public override string ToString() => "{Width=" + _width.ToString() + ", Height=" + _height.ToString() + "}";
```

Comment:
Yup got it... but, hmmm..., do I make this change in this PR since this is existing code...? Because I learnt over last few months that the scope of the changes in PR should be restricted to the issue... 🤔